modular_skyrat/master_files/code/datums/traits/negative.dm

```diff
@@ -8,6 +8,26 @@
 	medical_record_text = "Patient is incapable of communicating their emotions."
 	icon = "question-circle"
 
+/datum/quirk/fragile
+	name = "Fragility"
+	desc = "You feel incredibly fragile. Burns and bruises hurt you more than the average person!"
+	value = -6
+	mob_trait = TRAIT_FRAGILE
```

Comment:
Now that I think about it. What does this trait do? Does it do anything upon being added or removed? Because if not, I don't know what the point of having it apply a trait is.

Reply:
You are totally right actually, there are no references to this trait anywhere! Must have been when I was treading incredibly carefully when looking at other code to try and do the right thing. I'll go ahead and remove the define and the trait reference.